### ISSUE TITLE
Doesn't compile on ATmega32U4 (Micro, Leonardo)

### ISSUE THREAD

**Issue by adienakhmad** [SlashDevin/NeoSWSerial#3]
`error: 'TCCR2A' was not declared in this scope`
`error: 'TCCR2B' was not declared in this scope`

**Comment by SlashDevin** [comment]
Confirmed.

**Comment by SRGDamia1** [comment]
The 32u4 doesn't have a timer2.  Could one of the other timers be used?

**Comment by SlashDevin** [comment]
Yes, but I don't have time to work on this.  Choosing the timer, clock scaler (from baud rate) and testing is what needs to be done.  If you get something working, I'd be glad to fold it in!

**Comment by StackerDEV** [comment]
Somebody got it fixed?

**Comment by jpmeijers** [comment]
Can you perhaps remove `ATmega32U4 (Micro, Leonardo)` from the bottom of README.md, as it is a red herring that causes a lot of wasted time. We can add it back when this issue is fixed.

(I'll file a PR)

**Comment by AKopytenko** [comment]
Same problem. 
Please remove Micro from the supported boards block in README.MD
